app/app.js

```diff
@@ -428,7 +428,7 @@ angular.module('portainer', [
       }
     })
     .state('service', {
-      url: '^/service/:id/',
+      url: '^/service/:id/:page_anchor',
```

Comment:
@deviantony I do not fully understand what you mean,  just as an example, the url might be:
http://127.0.0.1:9000/#/service/m132l2db91jg31txe9i8s6f1u-service-tasks
If not, please provide an example.